src/compose/ComposeMenu.js

```diff
@@ -234,56 +227,58 @@ class ComposeMenuInner extends PureComponent<Props> {
   });
 
   render() {
-    const { expanded, insertVideoCallLink, onExpandContract } = this.props;
-    const numIcons =
-      2 + (Platform.OS === 'android' ? 1 : 0) + (insertVideoCallLink !== null ? 1 : 0);
+    const { expanded, insertVideoCallLink } = this.props;
+
+    const onOpenActionSheet = () => {
+      const options = [
+        'Upload a file',
+        'Upload a picture',
+        'Take a picture',
+        'Add a video call',
+        'Cancel',
+      ];
+      const cancelButtonIndex = options.length - 1;
+
+      this.props.showActionSheetWithOptions(
+        {
+          options,
+          cancelButtonIndex,
+        },
+        buttonIndex => {
+          if (buttonIndex === 0) {
```

Comment:
These repeated `if` statements look pretty fragile.

Instead, let's define an array (maybe call it `buttons`) that we use for two things at once:
- to compute `options`
- to give the right behavior based on the given `buttonIndex`, using that index to access an item in `buttons`

It could be an array of objects like `{ option, callback }`, or something more like we do in src/action-sheets/index.js.